src/frontend/e2e/utils/selectors.ts

```diff
@@ -0,0 +1,18 @@
+export const getByTestID = (id: string): string => `[data-test-id="${id}"]`;
+export const getByText = (text: string): string => `text=${text}`;
+export const getByClassName = (className: string): string => `css=${className}`;
+export const getByID = (id: string): string => `[id="${id}"]`;
+export const getByName = (name: string): string => `[name="${name}"]`;
+
+export const SELECTORS  = {
+    "DEV": {
+        USERNAME: "Input_Username",
+        PASSWORD: "Input_Password",
+        LOGIN_BTN: "Input.Button"
+    },
+    "STAGING": {
+        USERNAME: "Input_Username",
+        PASSWORD: "Input_Password",
+        LOGIN_BTN: "Input.Button"
+    }
```

Comment:
can you add a comment of when to use these? Also should we condense if both staging and dev have the same key values?